app/services/adapters/structs/crime_application.rb

```diff
@@ -14,6 +14,13 @@ def case
       def ioj
         Structs::InterestsOfJustice.new(interests_of_justice)
       end
+
+      def documents
+        # TODO: Filter out deleted documents
```

Comment:
Seems like we will not get back deleted documents, thus no need to filter, so we can remove this comment I introduced in previous refactor 😄